package.json

```diff
@@ -3,8 +3,10 @@
   "version": "1.1.0",
   "description": "Passport.js strategy for trusted HTTP headers",
   "main": "src",
+  "typings": "types",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha && dtslint types",
```

Comment:
I would prefer to keep `dtslint` and `mocha` separate so `npm test` only runs the tests (for a quicker turnaround in the common case of changing the logic).

Maybe `test` and `lint-types` as separate tasks, where the Travis script calls both in the "modern Node" branch?